service/controller/cluster_resource_set.go

```diff
@@ -727,27 +726,13 @@ func newClusterResourceSet(config clusterResourceSetConfig) (*controller.Resourc
 		}
 	}
 
-	handlesFunc := func(obj interface{}) bool {
-		cr, err := key.ToCluster(obj)
-		if err != nil {
-			return false
-		}
-
-		if key.OperatorVersion(&cr) == project.Version() {
-			return true
-		}
-
-		return false
-	}
-
 	initCtxFunc := func(ctx context.Context, obj interface{}) (context.Context, error) {
 		return controllercontext.NewContext(ctx, controllercontext.Context{}), nil
 	}
 
 	var resourceSet *controller.ResourceSet
 	{
 		c := controller.ResourceSetConfig{
-			Handles:   handlesFunc,
```

Comment:
When we use a label selector we do not need the handles func anymore. This also leads to reduced garbage in the operator logs. So far we saw all kinds of logs about runtime objects we are actually not interested in.